mlserver/rockcraft.yaml

```diff
@@ -0,0 +1,128 @@
+# Based on https://github.com/SeldonIO/MLServer/blob/1.3.5/Dockerfile
+# Operator version is 1.17.1 version for this ROCK can be found in
+# https://github.com/SeldonIO/seldon-core/blob/v1.17.1/operator/config/manager/configmap.yaml
+name: mlserver
+summary: An image for Seldon MLServer Huggingface
+description: |
+  This image is used as part of the Charmed Kubeflow product.
+version: 1.3.5_20.04_1 # <upstream-version>_<base-version>_<Charmed-KF-version>
+license: Apache-2.0
+base: ubuntu:20.04
+run-user: _daemon_
+services:
+  mlserver:
+    override: replace
+    summary: "mlserver service"
+    startup: enabled
+    command: bash -c 'export PATH=/opt/conda/bin/:/opt/mlserver/.local/bin:${PATH}:/usr/bin && export PYTHONPATH=/opt/mlserver/.local/lib/python3.8/site-packages/:${PYTHONPATH} && eval $(/opt/conda/bin/conda shell.bash hook 2> /dev/null) && mlserver start ${MLSERVER_MODELS_DIR}'
```

Comment:
why you need bash -c ? also put the PATH and PYTHONPATH into the env section of the service

Reply:
This is because of Conda. We use this method of executing commands in such cases. Here is rationale from other ROCK, I can add it here as well:
```
    # Yet again, use a subshell to jam conda into a working state. Can't use bashrc, because it immediately
    # exits if PS1 isn't set, so no-go from scripts.

```

Reply:
the export part not in env is also because of conda ?

Reply:
Yes.